migrate.py

```diff
@@ -8,6 +8,14 @@
 from traceback import print_exc
 from typing import List, Literal, NoReturn, Optional, Tuple
 
+def execute_sql(cursor, query, params=None):
+    if params:
+       #print("Executing SQL Query:", query, "with params:", params)
+        cursor.execute(query, params)
+    else:
+        #print("Executing SQL Query:", query)
+        cursor.execute(query)
```

Comment:
To make this a bit shorter, can you move this function to after the definition of `cursor`? That way it isn't a necessary argument. (Also please remove the commented out prints)